static/src/inline-svgs/acquisitions/apple-pay-mark.svg

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
I would run this through https://jakearchibald.github.io/svgomg/ - it can probably reduce the size by 50/60%

Reply:
nice idea, I tried the lowest setting 1.24kb but ended up with this:
![image](https://user-images.githubusercontent.com/7304387/51113935-b0af0100-17fb-11e9-9a58-426ef5eaaf1d.png)
so I tried the one above that 1.94kb and it's ok
![image](https://user-images.githubusercontent.com/7304387/51113955-bb699600-17fb-11e9-9dfd-4ffad6c3dbef.png)